network/src/commonMain/kotlin/com/wire/kalium/network/api/user/client/ClientResponse.kt

```diff
@@ -38,6 +38,6 @@ data class ClientsOfUsersResponse(
 @Serializable
 data class SimpleClientResponse(
     @SerialName("id") val id: String,
-    @SerialName("class") val deviceClass: String
+    @SerialName("class") val deviceClass: DeviceTypeDTO? = DeviceTypeDTO.Unknown
```

Comment:
We only need to do the same for `ClientDTO` since it can be null over there as well